test/performance/select_perf_test.lua

```diff
@@ -0,0 +1,176 @@
+local fio = require('fio')
+local fiber = require('fiber')
+local net_box = require('net.box')
+local log = require('log')
+
+local t = require('luatest')
+local g = t.group('perf')
+
+local helpers = require('test.helper')
+
+math.randomseed(os.time())
+
+g.before_all = function()
+    g.cluster = helpers.Cluster:new({
+        datadir = fio.tempdir(),
+        server_command = helpers.entrypoint('srv_select'),
+        use_vshard = true,
+        replicasets = {
+            {
+                uuid = helpers.uuid('a'),
+                alias = 'router',
+                roles = { 'crud-router' },
+                servers = {
+                    { instance_uuid = helpers.uuid('a', 1), alias = 'router' },
+                },
+            },
+            {
+                uuid = helpers.uuid('b'),
+                alias = 's-1',
+                roles = { 'customers-storage', 'crud-storage' },
+                servers = {
+                    { instance_uuid = helpers.uuid('b', 1), alias = 's1-master' },
+                    { instance_uuid = helpers.uuid('b', 2), alias = 's1-replica' },
+                },
+            },
+            {
+                uuid = helpers.uuid('c'),
+                alias = 's-2',
+                roles = { 'customers-storage', 'crud-storage' },
+                servers = {
+                    { instance_uuid = helpers.uuid('c', 1), alias = 's2-master' },
+                    { instance_uuid = helpers.uuid('c', 2), alias = 's2-replica' },
+                },
+            },
+            {
+                uuid = helpers.uuid('d'),
+                alias = 's-2',
+                roles = { 'customers-storage', 'crud-storage' },
+                servers = {
+                    { instance_uuid = helpers.uuid('d', 1), alias = 's3-master' },
+                    { instance_uuid = helpers.uuid('d', 2), alias = 's3-replica' },
+                },
+            }
+        },
+    })
+    g.cluster:start()
+end
+
+g.after_all = function()
+    g.cluster:stop()
+    fio.rmtree(g.cluster.datadir)
+end
+
+g.before_each(function() end)
+
+local function insert_customers(conn, id, count, timeout, report)
+    local customer = {id, box.NULL, 'David', 'Smith', 33, 'Los Angeles'}
+    local start = fiber.clock()
+
+    while (fiber.clock() - start) < timeout do
+        local ok, res, err = pcall(conn.call, conn, [[package.loaded.crud.insert]], {'customers', customer})
+        if not ok then
+            log.error('Insert error: %s', res)
```

Comment:
When error happen test prints messages like below:
```
Insert error: table: 0x1aa18998                                                                                                                               
Insert error: table: 0x1aa18c58                                                                                                                               
Insert error: table: 0x1aa18670                                                                                                                               
Insert error: table: 0x1aa18030                                                                                                                               
Insert error: table: 0x1aa182f0                                                                                                                               
Insert error: table: 0x1aa17d60
```
I suppose table address is not useful. It would be better to print error message to understand a reason of error.

Reply:
Currently it should be better. Fixed.